core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -174,10 +177,42 @@ public static void recordTaskExecutionTime(Task task, long duration, boolean inc
 			getTimer(classQualifier, "http_task_execution", "taskType", task.getTaskDefName(), "includeRetries", "" + includesRetries, "status", task.getStatus().name()).record(duration, TimeUnit.MILLISECONDS);
 			//counter(classQualifier, "http_task_execution", "taskType", task.getTaskDefName(), "status", task.getStatus().name());
 		}
-			
+
+		if (task.getTaskType().equals("WAIT")) {
+			Set<String> names = new HashSet<String>(Arrays.asList("waitchecksum", "waittranscode", "waittransfer", "waitsherlock", "episodicwaitpending", "waitpending"));
```

Comment:
Instead of defining the same Set in multiple methods, I'd recommend defining a private method to return it and call that method from the 3 methods in this file that define the Set. If that Set ever changes, we'll only need to make the change in one place.